ccm_web/server/localTypes/ltijs/index.d.ts

```diff
@@ -0,0 +1,274 @@
+/*
+Minimum temporary types for ltijs, building upon existing @types/ltijs and preliminary updates by the same author
+See https://github.com/paulschwoerer/DefinitelyTyped/commit/9e9bec14b02e3c78051c07862d2c305a7ac98c39
+This file does not include types for service classes (Grade, DeepLinking, NamesAndRoles, DynamicRegistration).
+
+References
+- https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html
+- https://stackoverflow.com/questions/45099605/ambient-declaration-with-an-imported-type-in-typescript
+*/
```

Comment:
I've continued to wrestle with the `ltijs` types here, but this is where I'm at for now. Instead of overriding or patching, I'm basically just including all the types I think we need for now (we may be even be able to manage with less). Things seem to be working. I will see if I can open a PR with a few changes either against the WIP version or DefinitelyTyped.

Does it seem okay to copy-and-paste and update these types here? Not sure if that's in violation of copyright or whatnot. The license for DT is MIT, though the original type author may hold some rights.

Reply:
How is this? https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/40/commits/1117466cf171c582b0076d502d299fbbedd8c8e3

I also wonder if I need to include anything about code patterns taken from ROHQ, specifically `usePromise` and `handleErrors`. I asked James about `usePromise`, he said using it was fine, I didn't need to mention him, but that project carries an Apache license. I'm new to re-purposing code like this.

Reply:
I feel Code share from ROHQ should be fine since it is internal project from ITS. Good you checked with ROHQ team members.

Reply:
Just checking to see if the documentation I provided in `localTypes/ltijs/index.d.ts` looks like it's enough to you, @pushyamig

Reply:
Looks Good to me.